CONTRIBUTING.md

```diff
@@ -35,6 +35,16 @@ Navigate into the repo:
 cd pynecone
 ```
 
+Install poetry:
+```
+pip install poetry
+```
+
+Install your local Pynecone build:
+```
+python -m pip install -e .
```

Comment:
Here we should run `poetry install` instead to install the Pynecone project requirements.